src/stripe/services/stripe.service.ts

```diff
@@ -36,6 +36,11 @@ export class StripeService {
     const userId = user.id
     const customerId = user.metaData?.customerId
 
+    // Filter out undefined values from metadata as Stripe doesn't accept them
+    const filteredMetadata = Object.fromEntries(
+      Object.entries(restMetadata).filter(([_, value]) => value !== undefined),
+    )
+
```

Comment:
If we end up w/ a consumer of this passing `undefined` values.. do we really want this error state to be swallowed?

Seems like we'd want Stripe to tell us if we're trying to send over these properties? 🤔